.circleci/config.yml

```diff
@@ -502,16 +533,43 @@ workflows:
           <<: *on-integ-and-version-tags
       - platform_build:
           <<: *context
-          <<: *on-integ-and-version-tags
+          <<: *on-any-branch # revert on-integ-and-version-tags
```

Comment:
need to actually revert

Reply:
will revert when I see all tests can pass